### DISCUSSION TITLE
How can I have an account transferred if something happens to me?

### DISCUSSION THREAD

**Discussion by Winter-JS** [vercel/community#4075] [Feature Question] [answered]
### Summary

Hello, I was having a discussion with a friend that I do development for and we host a few projects on Vercel however we have always just used my account. Justifying $20 a month isn't really possible considering the benefits aside from them having access are useless to us. 

If I were to suddenly die or were unable to continue development, is there a way for the account to be transferred to their email? 

### Steps to Reproduce

None general question about an account.

**Comment by amyegan** [comment, marked as answer]
In the event that access to your account was lost, your friend could deploy the code on their own account. If the project uses a third party custom domain that they own, then they can [verify domain access](https://vercel.com/docs/projects/domains/add-a-domain#verify-domain-access) with a TXT record to use it on their account.

Don't forget that Hobby accounts are restricted to non-commercial personal use as outlined in the [Fair Use Policy](https://vercel.com/docs/limits/fair-use-policy#).
